atlas-fastq-provider-functions.sh

```diff
@@ -456,6 +456,20 @@ function fetch_library_files_from_sra_file() {
             hash fastq-dump 2>/dev/null || { echo >&2 "The NCBI fastq-dump tool is required,  but it's not installed.  Aborting."; exit 1; }
             fastq-dump -I --split-files $library
             if [ $? -eq 0 ]; then
+                nlines=
+                lastfile=
+                ls *.fastq | while read -r l; do
+                    filelines=$(cat $l | wc -l)
```

Comment:
This could be just `filelines=$(wc -l $l)` but I guess that makes no difference?

Reply:
So the reason I use the command this way is:

```
> wc -l atlas-fastq-provider-functions.sh 
1141 atlas-fastq-provider-functions.sh
> cat atlas-fastq-provider-functions.sh | wc -l
1141
```

i.e. I'm trying to get the line number only. But I've committed a third way.